CollAction/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -118,6 +117,25 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("DataProtectionKeys");
                 });
 
+            modelBuilder.Entity("CollAction.Models.DonationEventLog", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd();
+
+                    b.Property<string>("EventData")
+                        .HasColumnType("json");
+
+                    b.Property<int>("Type");
+
+                    b.Property<string>("UserId");
```

Comment:
There is no maxlength here. Although it'll work fine, it might be wiser to add it here. Adding a MaxLenght or an index later will be troublesome on varchar(max) columns.

Reply:
It contains stripe json messages, no theoretical upper limit for that. I'm not sure what I'd set it to.